query-languages/m/m-spec-types.md

```diff
@@ -252,7 +252,7 @@ A _table-type value_ is used to define the structure of a table value.
 _table-type:_<br/>
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`table` _row-type<br/> 
```

Comment:
Isn't the row-type itself optional?

Reply:
By optional, do you mean that `type table` by itself is a valid type? If so, I think that case is covered by the `primitive-type` rule whereas `table-type` (this rule) covers the case where a table type is defined with columns being specified.

Reply:
Yes, that's what I meant. If it's covered elsewhere, then we're good.